### PR TITLE
Deployment

### PR DESCRIPTION
**Pull request by julianlen**
I created four different directories within the `env` directory.

base/ - It is for regtest, it has the same parameters as testnet with different chain id and web3_url

testnet/ & mainnet/  -These directories contain the configuration for deployment almost ready on each network. It only needs to set up the private keys. Before deploying on Mainnet, all the variables must be rechecked.

To easily find the addresses and private keys in the config/ files, I added the keywords <env_env> or <env_priv_key>.

zk/ - It contains the variable as those were on zkSync originally.

Most of the changes in the env variables were specified on the Aggregation document.

Also, I modified the hardhat.config.ts, so it reads the `proc.env.ETH_CHAIN_ID` rather than being hardcoded

`confirmations_for_eth_event` for mainnet was settled with 10 based on [this](https://discord.com/channels/722409280497516566/722409281130987522/833322379073421362)

### CODE REVIEW

**Review by raullaprida — CHANGES_REQUESTED** [review]
Small comments done.
Also, The purpose of the etc/env/zk folder is not clear to me. /base has the same reference values

File: etc/env/mainnet/contracts.toml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Addresses of the deployed zkSync contracts.
+# Values of this file are updated automatically by the contract deploy script.
+
+[contracts]
+UPGRADE_GATEKEEPER_ADDR="0x5E6D086F5eC079ADFF4FB3774CDf3e8D6a34F7E9"
```
[the remark this answers is not in the document]
Comment (julianlen) [reply etc/env/mainnet/contracts.toml:5]: Shouldn't be an issue, but change it just in case

[the remark this answers is not in the document]
Comment (julianlen) [reply etc/env/mainnet/contracts.toml:5]: These addresses are upgraded when the contracts are deployed

File: etc/env/testnet/contracts.toml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+[contracts]
+UPGRADE_GATEKEEPER_ADDR = "0x7389030eDb27924BCe611AdcC1D4b7dbbA2aEc84"
```
[the remark this answers is not in the document]
Comment (julianlen) [reply etc/env/testnet/contracts.toml:2]: These addresses are upgraded when the contracts are deployed

**Review by raullaprida — APPROVED** [review]
LGTM

**Pull request merged**